package.json

```diff
@@ -35,12 +35,14 @@
   "homepage": "https://github.com/zkochan/unpack-stream#readme",
   "dependencies": {
     "@types/node": "*",
+    "@types/ssri": "^6.0.1",
     "decompress-maybe": "^1.0.0",
     "ssri": "^6.0.0",
     "tar-fs": "^2.0.0"
   },
   "devDependencies": {
-    "@types/tape": "^4.2.30",
+    "@types/tape": "^4.2.33",
+    "@types/tar-fs": "^1.16.1",
```

Comment:
this should be a prod dep

Reply:
It doesn’t need to be a prod dependency, as none of the types from it are needed in the public type definition.

Unlike `ssri`, usage of `tar-fs` is merely an implementation detail.